Formula/openjdk@11.rb

```diff
@@ -46,15 +58,25 @@ class OpenjdkAT11 < Formula
     end
   end
 
+  patch do
+    # Fix for https://bugs.openjdk.java.net/browse/JDK-8266248 on Big Sur
+    url "https://github.com/openjdk/jdk11u-dev/commit/e44258cd04fb8d1ea727d322a0e661e44306ec57.patch?full_index=1"
+    sha256 "64ac56423da1d09013e4b14246fca60cb0551bda3fc2abcc23213e11f4ad709d"
+  end
+
   def install
-    framework_path = File.expand_path(
-      "../SharedFrameworks/ContentDeliveryServices.framework/Versions/Current/itms/java/Frameworks",
-      MacOS::Xcode.prefix,
-    )
+    framework_path = ""
+    on_macos do
+      framework_path = File.expand_path(
+        "../SharedFrameworks/ContentDeliveryServices.framework/Versions/Current/itms/java/Frameworks",
+        MacOS::Xcode.prefix,
+      )
+    end
```

Comment:
We could probably move this definition inside the next `on_macos` block.

Reply:
It looks like we need `framework_path` in both `on_macos` blocks below, so we'd have to define it twice.  Not sure if that is what we want here.

Reply:
It's currently defined in one `on_macos` block, so the definition can be seen across different ones.

Reply:
Ah, I did not realize the scoping worked like that.  In that case, I'll make the change.